db/database.go

```diff
@@ -61,11 +61,11 @@ var (
 	DefaultDeltaSyncRevMaxAge = uint32(60 * 60 * 24) // 24 hours in seconds
 )
 
-var DefaultCompactInterval = uint32(1440) // Default compact interval in minutes
+var DefaultCompactInterval = uint32(60 * 60 * 24) // Default compact interval in seconds = 1 Day
 
 const (
-	CompactIntervalMin = 60
-	CompactIntervalMax = 86400
+	CompactIntervalMinDays = float32(0.042) //1 Hour in days
```

Comment:
This value is rounded down slightly to `0.04` in CBS - anyone wanting to match that will hit a validation error. We should probably drop ours to match.

```suggestion
	CompactIntervalMinDays = float32(0.04) // ~1 Hour in days
```